frontend/public/components/utils/dropdown.jsx

```diff
@@ -346,8 +346,8 @@ export class Dropdown extends DropdownMixin {
         return;
       }
 
-      const selected = key === selectedKey;
-      const hover = key === keyboardHoverKey;
+      const selected = (key === selectedKey) && !this.props.noSelection;
+      const hover = (key === keyboardHoverKey) && !this.props.noSelection;
```

Comment:
We want to leave the hover style so keyboard navigation of the menu still works.

Reply:
It's only the selected style we would remove.